config/config.go

```diff
@@ -101,7 +102,7 @@ func (c Configuration) New(
 		return nil, nil, errors.New("no service name provided")
 	}
 	if c.Sampler == nil {
-		c.Sampler = &SamplerConfig{Type: samplerTypeProbabilistic, Param: 0.01}
+		c.Sampler = &SamplerConfig{Type: jaeger.SamplerTypeProbabilistic, Param: 0.001}
```

Comment:
Did you mean to change this magic number? I think it should also be a const declared somewhere